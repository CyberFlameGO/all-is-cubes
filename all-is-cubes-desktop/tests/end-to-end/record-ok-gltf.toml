bin.name = "all-is-cubes"
# Recording just the empty for now because that will be less exciting and bulky than
# actual meshes. Maybe revisit this later.
args = "--no-config-files --template blank --graphics record -o test-record-blank.gltf"
