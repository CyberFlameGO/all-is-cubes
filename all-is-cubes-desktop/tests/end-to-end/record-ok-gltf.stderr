...

Wrote test-record-blank.gltf
